doc/whats_new/v0.24.rst

```diff
@@ -368,6 +368,11 @@ Changelog
   `predict` and `score` would fail when `fit_intercept=False` and there was
   one feature during fitting. :pr:`18121` by `Thomas Fan`_.
 
+- |Fix| Added the missing `l1_ratio` parameter in
+  :class:`linear_model.Perceptron`, to be used when `penalty='elasticnet'`.
+  This changes the default from 0 to 0.15. :pr:`18622` by
+  :user:`Haesun Park <rickiepark>`.
```

Comment:
You need to add the suggestion by @NicolasHug 

```
We will also need the following entry at the top of the whatsnew, under the "changed models" section:

```
        |Fix| ::class:linear_model.Perceptron when penalty='elasticnet'
```

On the top of the file around l. 25 "Changed models"

Reply:
Thanks, I forgot it